src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf

```diff
@@ -298,6 +298,22 @@
                 <source>The image is portrait oriented ({{ width }}x{{ height }}px). Portrait oriented images are not allowed.</source>
                 <target>D'Bild ass am Héichformat ({{ width }}x{{ height }}px). Biller am Héichformat sinn net erlaabt.</target>
             </trans-unit>
+            <trans-unit id="78">
+                <source>An empty file is not allowed.</source>
+                <target>En eidele Fichier ass net erlaabt.</target>
+            </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Den Domain-Numm konnt net opgeléist ginn.</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Dëse Wäert entsprécht net dem erwaarten Zeechesaz {{ charset }}.</target>
+            </trans-unit>
```

Comment:
IIRC we merged all translations into `2.3` in the past to make contributing for translators easier.